playbooks/roles/topbeat/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+topbeat_app_dir: "{{ COMMON_APP_DIR }}/topbeat"
```

Comment:
Per discussion, we won't combine at this time.
I'll delete the comment for now.